RELEASES_CHECKLIST.md

```diff
@@ -0,0 +1,45 @@
+# How to release
+
+Obviously there has to be an ink! PR, it has to be approved by core team
+members, and the CI has to be green.
+
+Generally, we bump and publish the versions of all crates in this
+mono-repository in lockstep.
+It does not matter if it's a `PATCH`, `MINOR`, `MAJOR` release.
+
+Reasons for doing it this way:
+* Keeps our [`RELEASES.md`](https://github.com/paritytech/ink/blob/master/RELEASES.md)
+  simple and easy to grasp.<br>
+  We can still bundle all changes there together and don't have to go to a
+  more fine-grained crate level.
+* Easier to follow reported issues.<br>
+  All crates are closely tied together anyway and if someone reports an issue with
+  e.g. `3.1` we know what they mean.
+* Easier to keep the workshop/documentation/playground/user interfaces in sync.
+* Easier for users to follow changes.<br>
+  Those will all be listed in one section of the changelog, bundled together,
+  released at the same time.
+* Third party tooling like dependabot can easily extract the changelog.
+
+## Checklist
+
+1. Make sure you've moved the changes in the `RELEASES.md` from `[Unreleased]` into a new section for the release.
```

Comment:
```suggestion
1. Make sure you've moved the changes in the `RELEASES.md` from `[Unreleased]` into a new section for the release.
1. Check that all PRs since the last release are now in the new release section, in case the `[Unreleased]` section was incomplete.
```